### PR TITLE
[plugin.video.funkmediathek] v1.2.0

### PR DESCRIPTION
**Pull request by prof-membrane**
### Description
This is an add-on for www.funk.net. 

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.funkmediathek/addon.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+		<import addon="xbmc.python" version="2.25.0"/>
+		<import addon="script.module.libmediathek3" version="1.3.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+		<license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+		<source>http://github.com/someone/myaddon</source>
```
Comment (phil65) [inline plugin.video.funkmediathek/addon.xml]: this looks wrong :)

File: plugin.video.funkmediathek/resources/lib/jsonparser.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	return d
+	
+def _cleanPlot(plot):
+	plot = plot.replace('<p>','')
+	plot = plot.replace('</p>','')
+	plot = plot.replace('<h4>','')
+	plot = plot.replace('</h4>','')
+	while '  ' in plot:
+		plot = plot.replace('  ',' ')
+	plot = plot.replace('\n ','')
```
Comment (phil65) [inline plugin.video.funkmediathek/resources/lib/jsonparser.py]: replacing with [CR] would keep the line breaks (not sure if that is wanted)

Comment (prof-membrane) [inline plugin.video.funkmediathek/resources/lib/jsonparser.py]: I guess I could take that out. This was used to clean up some messy plots with line breaks all over the place.

File: plugin.video.funkmediathek/addon.xml (hunk 1 above)
Comment (prof-membrane) [inline plugin.video.funkmediathek/addon.xml]: Oooops.

**Comment by MartijnKaijser** [comment]
@prof-membrane could you please change this for next update?
WARNING: Using the old language directory structure, please move to the new one.
See:
http://kodi.wiki/view/language_support#Layout_of_the_language_folders

**Pull request merged**